lib/model/query/entities.js

```diff
@@ -478,7 +478,7 @@ const _processSubmissionEvent = (event, parentEvent) => async ({ Datasets, Entit
   const { submissionId, submissionDefId } = event.details;
   const forceOutOfOrderProcessing = parentEvent?.details?.force === true;
 
-  const form = await Forms.getByActeeId(event.acteeId);
+  const form = await Forms.getByActeeId(event.acteeId, Form.PublishedVersion); // this appears to need a form definition... but why?
```

Comment:
I don't think the entity code below really needs this form to have a def? Does leaving out `Form.PublishedVersion` break something?

Looking at the other places in the code `Forms.getByActeeId` is used, those don't seem to use a form def either or make use of the `versionJoinCondition`? Maybe `getByActeeId` was built to be super general (get the current draft or published def) but was never needed.